code/game/area/areas/ruins/space.dm

```diff
@@ -168,7 +168,8 @@
 /area/ruin/space/has_grav/deepstorage/crusher
 	name = "\improper Deep Storage Recycler"
 
-
+/area/ruin/space/has_grav/deepstorage/pharmacy
+	name = "\improper Deep Storage Pharmacy"
```

Comment:
```suggestion
	name = "\improper Deep Storage Pharmacy"
	
```
New line after 172 to break it off from the following line `//Ruin of Abandoned Zoo`

Reply:
Looks to be good, thanks!